tests/test_delete_company.py

```diff
@@ -0,0 +1,20 @@
+"""This module allows you to delete company and check if the company is absent"""
+import allure
+from base import COMPANY_EXISTS_URL, DELETING_OF_COMPANY
+
+
+@allure.feature('Deleting a company')
+def test_delete_company(app):
+    """Deleting of the company"""
+
+    with allure.step("Login"):
+        app.authentication("cowner@gmail.com", "cowner")
```

Comment:
why username and password are there like strings?

Reply:
I have changed it